modules/Cyaxaress/User/Database/Migrations/2020_07_01_000000_create_users_table.php

```diff
@@ -34,7 +34,7 @@ public function up()
             $table->rememberToken();
             $table->timestamps();
 
-            $table->foreign('image_id')->references('id')->on('media')->onDelete('SET NULL');
+            $table->foreign('image_id')->references('id')->on('users')->onDelete('SET NULL');
```

Comment:
Its a relationship between users table and mrdia table, not users table with itself. 
So it's not current